daps-xslt/asciidoc/postprocess.xsl

```diff
@@ -170,4 +170,12 @@
     </xsl:if>
   </xsl:template>
 
+  <xsl:template match="d:formalpara[d:para/d:screen]">
```

Comment:
I think in the examples, we already had an `<informalexample>` wrapping around this, so I think you should probably match on `informalexample[formalpara]` rather than this.

Reply:
Hmn. :thinking:  I think we need both. Depending on how the author wrote the AD file, we can't be sure we get always an `informalexample`.

I've added both cases to be sure.

Reply:
I saw these occurrences in previous commits. As I said, it's a matter of how the authors write the AD source. Therefore, I think both rules would be necessary. That rule shouldn't hurt.